apps/asap-server/test/handlers/invalidate-cache/index-handler.test.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  CloudFrontClient,
+  CreateInvalidationCommand,
+} from '@aws-sdk/client-cloudfront';
+import { createHandler } from '../../../src/handlers/invalidate-cache/index-handler';
+const mockSend = jest.fn();
+
+jest.mock('@aws-sdk/client-cloudfront', () => ({
```

Comment:
i think we generally use the DI in asap-server instead of jest.mock, proxyquire etc but I guess its fine.